specs/latest/1.0/index.html

```diff
@@ -1690,9 +1689,8 @@ <h2 class="no-toc">Table of contents</h2>
                            GLenum srcAlpha, GLenum dstAlpha);
 
     void bufferData(GLenum target, GLsizeiptr size, GLenum usage);
-    void bufferData(GLenum target, ArrayBuffer? data, GLenum usage);
-    void bufferData(GLenum target, ArrayBufferView data, GLenum usage);
-    void bufferSubData(GLenum target, GLintptr offset, BufferDataSource data);
+    void bufferData(GLenum target, [AllowShared] BufferSource? data, GLenum usage);
```

Comment:
I assume that passing null for this union type to bufferData won't cause a failure to resolve the overload -- that it will still generate an INVALID_OPERATION OpenGL error per the spec.